src/longopt/psconvert_inc.h

```diff
@@ -25,74 +25,45 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'H', "",
-	          "",                  "",
+	{ 0, 'A', "adjust|autocrop|crop",
 	          "",                  "",
+	          "r,u",               "round,no_timestamp",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'C', "gs_option|ghost_option", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'D', "outdir|out_dir",    "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'E', "dpi",               "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'F', "outfile|out_name|prefix", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "gs_path|ghost_path", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'H', "scale",             "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'I', "resize",
 	          "",                  "",
+	          "m,s,S",             "margins,size,scale",
 		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
+	{ 0, 'L', "listfile|list_file", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'M', "pslayer|ps_layer",
+	          "b,f",               "bg|background,fg|foreground",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'N', "bb_style|show_bbox",
```

Comment:
I prefer `bgcolor`. This is mainly not about the BoundingBox per se but specially on the possibility to paint the BB area.